tests/testthat/test-object-package.R

```diff
@@ -0,0 +1,34 @@
+context("Object: Package")
+
+test_that("author with one given name ", {
+  expect_equal(author_desc(list(given="Given",
+                                family="Family",
+                                role=c("aut", "cre"),
+                                email="given.family@email.tld",
+                                comment=NULL)),
+               "Given Family \\email{given.family@email.tld}"
+  )
+})
+
+test_that("author with more than one given name ", {
+  expect_equal(author_desc(list(given=c("First", "Second"),
```

Comment:
Can you construct with `person()`? That should make simpler as you won't need to supply all the arguments.

Reply:
Done